src/main/java/uz/enterprise/mytex/entity/Group.java

```diff
@@ -0,0 +1,29 @@
+package uz.enterprise.mytex.entity;
+
+import lombok.*;
+import uz.enterprise.mytex.entity.audit.Auditable;
+
+import javax.persistence.*;
+
+/**
+ * @author - 'Zuhriddin Shamsiddionov' at 10:33 AM 10/24/22 on Monday in October
+ */
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "groups")
+public class Group extends Auditable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, name = "name")
```

Comment:
```suggestion
    @Column(name = "name", nullable = false)
```
follow the order of the paramteres when using @Column annotation
1. column name
2. validation params